honeybee_radiance/postprocess/en17037.py

```diff
@@ -0,0 +1,264 @@
+"""Functions for post-processing EN 17037 daylight outputs."""
+import json
+import os
+
+from .annual import filter_schedule_by_hours, _process_input_folder
+
+
+def _metrics(values, occ_pattern, threshold, total_hours):
+    """Calculate annual metrics for a sensor.
+
+    Args:
+        values: Hourly illuminance values as numbers.
+        occ_pattern: A list of 0 and 1 values for hours of occupancy.
+        threshold: Threshold value for daylight autonomy.
+        total_occupied_hours: An integer for the total number of occupied hours,
+            which can be used to avoid having to sum occ pattern each time.
+
+    Returns:
+        daylight autonomy
+    """
+    def _percentage(in_v, occ_hours):
+        return round(100.0 * in_v / occ_hours, 2)
+
+    da = 0
+    for is_occ, value in zip(occ_pattern, values):
+        if is_occ == 0:
+            continue
+        if value > threshold:
+            da += 1
+
+    return _percentage(da, total_hours)
+
+
+def metrics_to_files(
```

Comment:
I know that you are trying to follow the example of annual-daylight but it might be a good idea to name this method with a slightly different name just to avoid one method overwriting another if they are both imported to the same place. Maybe we should call this `en17037_metrics_to_files`